lib/web_ui/lib/src/engine/skwasm/skwasm_impl/raw/text/raw_paragraph.dart

```diff
@@ -72,6 +72,24 @@ external int paragraphGetPositionForOffset(
   Pointer<Int32> outAffinity,
 );
 
+@Native<Bool Function(ParagraphHandle, Float, Float, Pointer<Float>, Pointer<Uint32>, Pointer<Bool>)>(symbol: 'paragraph_getClosestGlyphInfoAtCoordinate')
+external bool paragraphGetClosestGlyphInfoAtCoordinate(
+  ParagraphHandle handle,
+  double offsetX, double offsetY,
+  Pointer<Float> graphemeLayoutBounds,   // 4 floats, [LTRB]
```

Comment:
You can do `Pointer<RawRect>` here, which is the same type but it's just a little more expressive.